docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst

```diff
@@ -234,15 +224,14 @@ the original `CreatedEvent` (see above):
 
 - **template_id** - The contract's template id.
 - **contract_id** - The contract id.
-- **arguments** - The contract's create arguments. This field is a protobuf ``oneof``
-  and it allows either passing the contract's create arguments typed (as ``create_arguments``)
-  or as a byte array (as ``create_arguments_blob``).
-  Generally, clients should use the ``create_arguments_blob`` for convenience since they can be received as such
-  from the stakeholder off-ledger (see above).
-- **metadata** - The contract metadata. This field can be populated as received from the stakeholder (see below).
+- **created_event_blob** - The contract's representation as an opaque blob encoding. This field is populated **only** on demand for ``GetTransactions`` and ``GetTransactionTrees`` streams (read more about :ref:`configuring transaction filters <transaction-filter>`).
```

Comment:
One part of this paragraph belongs in the previous chapter where you say that the contracts can be fetched in the LAPI queries

Reply:
Adapted 👍